web/client/components/geostory/layouts/sections/enhancers/currentPageSectionManager.jsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { compose, createEventHandler, mapPropsStream } from 'recompose';
+import { get, maxBy } from "lodash";
+import { Observable } from "rxjs";
+
+// TODO: externalize
+import { setObservableConfig } from 'recompose';
```

Comment:
mmmh, we need to externalize it one day or the other ;)

Reply:
yes, it should be done once for all, together with other libs an setups.